WordPress/Classes/ViewRelated/People/InvitePersonViewController.swift

```diff
@@ -172,7 +172,9 @@ class InvitePersonViewController : UITableViewController {
             return
         }
 
-        roleViewController.mode = .Static(roles: Role.inviteRoles)
+        let roles = (blog.siteVisibility == .Private) ? Role.inviteRolesForPrivateSite : Role.inviteRoles
```

Comment:
Not ver important but doing `let roles: Role = ...` would let you remove the double `Role.` at the end.